File: Laba1/Laba1/petrushka.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Laba1
{
    class petrushka : Washable
    {
        private int dirty = 10;

        public int Dirty { set { if (value > -1 && value < 11) dirty = value; } get { return dirty; } }
    }
}

File: Laba1/src/main/Petrushka.java
package main;

public class Petrushka {

    private int dirty = 10;

    public int getDirty() {
        return dirty;
    }

    public void setDirty(int dirty) {
        if(dirty>-1 && dirty<11) {
            this.dirty = dirty;
        }
    }
}
